php-classes/Slate/CBL/Calculators/Growth/PerformanceLevelMinusBaseline.php

```diff
@@ -8,7 +8,13 @@ class PerformanceLevelMinusBaseline implements IGrowthCalculator
 {
     public static function calculateGrowth(StudentCompetency $StudentCompetency)
     {
-        $baseline = $StudentCompetency->BaselineRating;
+        $baseline = round($StudentCompetency->BaselineRating, 1);
+        $performanceLevel = round($StudentCompetency->getDemonstrationsAverage(), 1);
```

Comment:
@nbey let's move the rounding into `getDemonstrationsAverage`, and put a public static config on `StudentCompetency` for the precision, defaulting to `1`. It seems like varying precisions are causing issues for schools and that it would be best for the delivered value to start with consistent rounding.